src/index.tsx

```diff
@@ -39,30 +39,39 @@ export type ErrorType =
   | 'invalidUserInput'
   | 'invalidLinkToken';
 
+/**
+ * @deprecated The type should not be used as it clashes with the native JS `Error` object.
+ * You should use `PinwheelError` instead. `Error` will be removed in version 2.4
+ */
 export type Error = {
   type: ErrorType;
   code: string;
   message: string;
   pendingRetry: boolean;
 };
 
-type EventPayload =
+// Export `Error` as `PinwheelError` to avoid native `Error` namespace clash
+export type PinwheelError = Error
```

Comment:
Do you think we should make `PinwheelErrorType` to match?

Reply:
i been thinking about this-- i can go either way with this...

Reply:
If I were starting brand new, I would probably name it `PinwheelErrorType`, but since it's already in there, maybe it's not worth it

Reply:
Yea, let's do that and follow the same playbook with marking `ErrorType` as deprecated